### PR TITLE
➕ add stylelint

### PR DESCRIPTION
**Pull request by RonjaMeinecke**
• add style lint `npm install --save-dev stylelint stylelint-config-standard`
https://stylelint.io/user-guide/get-started
• add `.stylelintrc.json` to root
• add stylelint script to `.package.json`
• add stylelint/prettier config
https://github.com/prettier/stylelint-config-prettier

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "extends": ["stylelint-config-prettier"]
```
Comment (fre-ben) [inline .stylelintrc.json]: "stylelint-config-standard" seem to be missing in your extends.

**Review by fre-ben — CHANGES_REQUESTED** [review]

File: .stylelintrc.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+}
```
Comment (HorstFrank) [inline .stylelintrc.json]: {
  "extends": ["stylelint-config-standard", "stylelint-config-prettier"]
}

File: package.json [hunk 3]
```diff
@@ -7,7 +7,8 @@
     "build": "next build",
     "start": "next start",
     "eslint": "eslint .",
-    "prettify": "prettier --write ."
+    "prettify": "prettier --write .",
+    "stylelint-check": "stylelint-config-prettier-check"
```
Comment (lmachens) [inline package.json]: There is a misunderstanding with this script. You said:

> • npm run stylelint-check to check if it is working

This script doesn't check if you have style issues. You can verify it by adding an invalid css property and run that script.

A script to check for style issues is missing.

**Review by lmachens — CHANGES_REQUESTED** [review]
gj

**Comment by DNaunin** [comment]
LGTM! 🚀

**Review by christophvogel — APPROVED** [review]
GJ 👍

File: .stylelintrc.json (hunk 2 above)
Comment (RonjaMeinecke) [reply .stylelintrc.json]: thanks ☺️

**Review by fre-ben — APPROVED** [review]
GJ! 💯

File: .stylelintrc.json (hunk 1 above)
Comment (RonjaMeinecke) [reply .stylelintrc.json]: thx 💪

**Pull request merged**